### ISSUE TITLE
continuous_freq causes Lisp error given rational numbers

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#785]
Imported from SourceForge on 2024-07-03 11:26:21
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2020-12-04 19:30:15
Original: https://sourceforge.net/p/maxima/bugs/3685

---

After `load(descriptive)`:
```{text}
(%i28) continuous_freq ([1,2,3,2,3,4,3,7/2] );

Maxima encountered a Lisp error:

 The value
   ((RAT SIMP) 7 2)
 is not of type
   REAL
```
I guess that there is an assumption that the inputs are only floats or integers. Should relax it to include at least rationals. Not sure about symbol numbers such as `%pi`. Maybe just leave that out for now.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-03 11:26:23
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2020-12-10 19:25:26
Original: https://sourceforge.net/p/maxima/bugs/3685/#6b09

---

- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-03 11:26:27
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2020-12-10 19:25:26
Original: https://sourceforge.net/p/maxima/bugs/3685/#7916

---

Fixed by commit a045616.
